### ISSUE TITLE
Improve contrast on deactivated user for Manage Users page

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5116]
## Area or Component
Go to Settings > Manage Users > (then select a deactivated user)

## Quick Summary
Contrast between the text/buttons for a deactivated user and the background is low which impacts readability

## Accessibility Violation
[All elements must have sufficient contrast](https://dequeuniversity.com/rules/axe/4.4/color-contrast-enhanced)

## Action Requested
Make the text/buttons for the deactivated user darker to improve contrast

## Screenshots (optional)
![Screen Shot 2023-01-25 at 11.30.49.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/a34ef010-7b4e-4232-a6df-a75713f2d8ea)

## Source Code (optional)

[which file or line is the issue attached to?]: #

## Original Deque Issue
https://dequeuniversity.com/rules/axe/4.6/color-contrast?application=AxeChrome

**Comment by BobanL** [comment]
If we follow the same ratio for contrast, from `adadad` to `6e6e6e`, it almost becomes more difficult to tell the difference between the disabled ones and not disabled ones. I'm not sure if that's a problem.

Original:
<img width="835" alt="image" src="https://user-images.githubusercontent.com/10108172/218851987-5d122208-c6eb-4e2c-84fa-118f01435033.png">
Example with OK contrast: 
<img width="884" alt="image" src="https://user-images.githubusercontent.com/10108172/218852206-43357f73-d2c9-4b66-964d-611a3ba82b11.png">

**Comment by BobanL** [comment]
@kenieh - Does the above change look OK to increase the contrast for the disabled fields?

**Comment by kenieh** [comment]
@BobanL Nice catch! LGTM.
